tests/model/test_operations.py

```diff
@@ -135,6 +136,33 @@ def draws_at_value(mean, std, cnt):
     return norm.isf(np.linspace(0.01, 0.99, cnt), loc=mean, scale=std)
 
 
+def test_var_grid_iterator():
+    # parent rate 0, parent rate 1,
+    # child 1 rate 0, child 2 rate 0, with shared smooth grid
+    # child 1 rate 1, different smooth grid.
+    # Note there can be two rates with the _same smooth id_ but different nodes.
+    var_df = pd.DataFrame(dict(
+        var_id=[0, 1, 2, 3, 4, 5, 6, 7],
+        smooth_id=[0, 1, 2, 2, 3, 4, 5, 6],
+        var_type=["rate", "rate", "rate", "rate", "rate", "mulcov_meas_value",
+                  "mulcov_rate_value", "mulcov_meas_std"],
```

Comment:
Should rate_value be in here too?

Reply:
I think "mulcov_rate_value" is what you call rate value. Meaning, the db uses "mulcov_rate_value" but we call them "rate_value" in another part of the DB.